Sources/TokamakStaticHTML/Shapes/Path.swift

```diff
@@ -18,13 +18,17 @@ import TokamakCore
 
 extension Path: ViewDeferredToRenderer {
   // TODO: Support transformations
-  func svgFrom(storage: Storage,
-               strokeStyle: StrokeStyle = .init(lineWidth: 0,
-                                                lineCap: .butt,
-                                                lineJoin: .miter,
-                                                miterLimit: 0,
-                                                dash: [],
-                                                dashPhase: 0)) -> AnyView {
+  func svgFrom(
```

Comment:
<!--
  1 failure:  Function body sho...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakStaticHTML/Shapes/Path.swift;
  Line: 21;
-->
- :no_entry_sign: Function body should span 50 lines or less excluding comments and whitespace: currently spans 52 lines (`function_body_length`)